### ISSUE TITLE
UnicodeEncodeError in yowsup-cli with Python 2.7.13 when recieving a non-ASCII character

### ISSUE THREAD

**Issue by DagoW** [tgalal/yowsup#1942]
I know that Unicode errors have been discussed in previous issues,
but even after reading them, I couldn't figure out how to fix the
problem by myself.

Whenever I receive a smiley or international character, yowsup-cli errors out
with this kind of message

```
  File "/home/user/git/yowsup2017/yowsup/demos/cli/layer.py", line 512, in onMessage
    MESSAGE_ID = message.getId()
UnicodeEncodeError: 'ascii' codec can't encode character u'\U0001f60a' in position 0: ordinal not in range(128)
```

Can somebody share his fixed version or explain how to do it?

Also, isn't this such a common scenario that everybody should get this error
sooner or later? If so, why not put the fix on the main branch in the repository?

Thanks, Dago

**Comment by DagoW** [comment]
OK guys, I found a working fix in #1646
Line 511 of yowsup/demos/cli/layer.py should be
`            MESSAGE = messageOut.encode('latin-1').decode() if sys.version_info >= (3, 0) else messageOut.encode('utf-8'),`

@tgalal are you going to fix this in the repository or should I just close this issue?

**Comment by tgalal** [comment]
fixed
